notebooks/openeo_vito_example.ipynb

```diff
@@ -75,17 +92,25 @@
     "process_graph = json.loads(sen2cor_masked.to_json())[\"process_graph\"]"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "4689a33e",
+   "metadata": {},
+   "source": [
+    "Convert process graph into an evalscript and print info about the created evalscript. The 'invalid_node_id' tells us up to which node the process graph could be converted. If it's \"None\", the whole process graph could be converted into an evalscript."
```

Comment:
Again we could use the `jsbeautifier` here, as we do in the NDVI evalscript.

Reply:
Added